service/src/main/java/org/cbioportal/service/exception/CacheOperationException.java

```diff
@@ -0,0 +1,7 @@
+package org.cbioportal.service.exception;
+
+public class CacheOperationException extends Exception {
```

Comment:
Love properly named exceptions instead of just using RuntimeException everywhere. Nice job.

Reply:
Thanks :)

Reply:
Btw I reworked the exception handling somewhat. While working on Spring Boot cBioPortal I came across class GlobalExceptionHandler that has the responsibility to handle problems. This now includes:

```
    @ExceptionHandler(CacheOperationException.class)
    public ResponseEntity<ErrorResponse> handleCacheOperationException(CacheNotFoundException ex) {
        ErrorResponse response = new ErrorResponse("Error evicting caches. Please try again or validate correct operation of your configured caching implementation.");
        return new ResponseEntity<>(response, HttpStatus.INTERNAL_SERVER_ERROR);
```